From ab80da4ea9ea88cdd802f80955e7382bd9368247 Mon Sep 17 00:00:00 2001
From: Benjamin Gill <github@bgill.eu>
Date: Sat, 16 Jun 2018 12:59:44 +0100
Subject: [PATCH] Fix new renamed_and_removed_lints warning (#247)

I've verified that this now produces no warnings with Rust 1.26.1 and
1.28.0-nightly (c3b09c968 2018-05-27)
---
 src/error_chain.rs           | 4 ++--
 src/impl_error_chain_kind.rs | 4 ++--
 src/lib.rs                   | 1 -
 3 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/src/lib.rs b/src/lib.rs
index d0881fcef1c1..6421194f8ca8 100644
--- a/vendor/error-chain-0.11.0/src/lib.rs
+++ b/vendor/error-chain-0.11.0/src/lib.rs
@@ -1,5 +1,4 @@
 #![deny(missing_docs)]
-#![allow(unknown_lints)] // to be removed when unused_doc_comments lints is merged
 #![doc(html_root_url = "https://docs.rs/error-chain/0.11.0")]
 
 //! A library for consistent and reliable error handling
-- 
2.17.1

